cmd/rook/discover.go

```diff
@@ -32,10 +32,14 @@ var (
 
 	// interval between discovering devices
 	discoverDevicesInterval time.Duration
+
+	// Uses ceph-volume inventory to extend devices information
+	usesCVInventory bool
 )
 
 func init() {
 	discoverCmd.Flags().DurationVar(&discoverDevicesInterval, "discover-interval", 60*time.Minute, "interval between discovering devices (default 60m)")
+	discoverCmd.Flags().BoolVar(&usesCVInventory, "use-cv", false, "Use ceph-volume inventory to extend storage devices information (default false)")
```

Comment:
please spell it out: `use-ceph-volume`. Then it's obvious that other storage providers don't want to pass it.

Reply:
done